packages/ui/src/components/Organisms/PageContent/BlockCta.tsx

```diff
@@ -1,7 +1,37 @@
-import { BlockCtaFragment } from '@custom/schema';
+import { BlockCtaFragment, CtaIconPosition, CtaIconType } from '@custom/schema';
+import clsx from 'clsx';
 import React from 'react';
 
 export function BlockCta(props: BlockCtaFragment) {
-  console.log(props);
-  return <div />;
+  return (
+    <a
```

Comment:
Is there a particular reason we are using the `<a>` tag instead of our custom `<Link>` component from `@custom/schema`?

Reply:
good point! Now using `Link`